guides/common/modules/proc_creating-a-personal-access-token.adoc

```diff
@@ -21,11 +21,10 @@ You can click *Copy to clipboard* to copy your {PAT}.
 
 .Verification
 . Make an API request to your {ProjectServer} and authenticate with your {PAT}:
-// tags in combination with `none` substitution have to be used here to render colon between USERNAME and PAT as normal.
 +
-[options="nowrap", subs="none,attributes"]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# curl https://<em>{foreman-example-com}</em>/api/status --user <em>USERNAME</em>:<em>PERSONAL_ACCESS_TOKEN</em>
+# curl https://_{foreman-example-com}_/api/status --user _USERNAME_:__PERSONAL_ACCESS_TOKEN__
```

Comment:
```suggestion
# curl https://_{foreman-example-com}_/api/status --user _My_Username_:__My_{PAT}__
```

Maybe try using the attribute now?

Reply:
I couldn't use attribute here due to missing underscores but I did reword it in this fashion.